src/backtrace.c

```diff
@@ -150,8 +148,12 @@ void backtrace_gdb(void) {
     const char fmt[] = "%s -nx -batch -x %s -p %d";
     static char cmd[sizeof(fmt) + 3 * MAX_BT_PATHSIZE];
     char filename[MAX_BT_PATHSIZE];
-    const char *gdb = (access(GDB_PATH, X_OK) ? "gdb" : GDB_PATH);
     int rc;
+#ifdef GDB_PATH
+    const char *gdb = (access(GDB_PATH, X_OK) ? "gdb" : GDB_PATH);
+#else
+    const char *gdb = "gdb";
```

Comment:
@jdinan In this case, the system path might not have the "gdb" in it. Still executing with "gdb" will result in an error. Are we still going to proceed with this? Maybe I am not getting something here?

Reply:
That's what I was thinking.  We could write some code to check if GDB is in the path.  Otherwise, you get something like this:
```
$ SHMEM_BACKTRACE=gdb test/unit/global_exit
sh: gdb: command not found
[0000] ERROR: ../../src/backtrace.c:175: backtrace_gdb
[0000]        Error generating backtrace using gdb (32512)
PE 0 called shmem_global_exit with status 0
```

Reply:
Ok. But, we are already checking that in the configure, right? If it is not in the path, we were not defining "GDB_PATH"

Reply:
Yeah, there is definitely overlap.  If I had to pick one of these, I think I would choose to use the runtime path to find gdb rather than what was found by configure

Reply:
Ok. Will take a look at this as a separate issue.